src/index.js

```diff
@@ -0,0 +1,48 @@
+import './index.css';
+import { postData, getData } from './modules/getapi.js';
+
+const form = document.getElementById('add-form');
+const container = document.getElementById('scores-container');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const refreshBtn = document.getElementById('refresh-btn');
+
+const displayScore = (person, personScore) => {
+  const tableRow = document.createElement('tr');
+  tableRow.classList.add('person-1');
+  tableRow.innerHTML = `
+   <td class="title">${person}</td>
+   <span> : </span>
+   <td class="scores">${personScore}</td>`.trim();
+
+  container.appendChild(tableRow);
+};
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const person = name.value;
+  const personScore = score.value;
+  await postData({
+    user: person,
+    score: Number(personScore),
+  });
+
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
```

Comment:
[ Optional Changes ]
I noticed that for every form submission, the data is prefetched and the list tends to get duplicated.
- [ ] It would be nice if you could remove this code, then only clear the input fields after form submission. And then one can only see the list when the refresh button is clicked. You could add these lines of code after posting the data in order to clear the input fields instead of displaying the data. 👍 

```javascript
  name.value = "";
  score.value = "";
```

I tested it and it works fine. The list only shows when I click on the refresh button and that's more efficient. 👍